package.json

```diff
@@ -2,9 +2,9 @@
   "name": "eslint-config-incuna",
   "version": "1.0.0",
   "description": "Incuna eslint shared config",
-  "main": "index.js",
+  "main": "eslintrc.json",
   "scripts": {
-    "test": "node index.js"
+    "test": "node eslintrc.json"
```

Comment:
Hmm but that's even more confusing. I think I either like `index.js` or `eslintrc.json` since it would be linting settings for this repo had we had any js files